app/views/talks/_talk.html.erb

```diff
@@ -22,7 +22,17 @@
                  youtube_id: talk.video_id} %>
   </div>
 
-  <div class="py-4 flex flex-col gap-4 my-4">
+  <div class="py-4 flex flex-col gap-4">
```

Comment:
The indentation of this partial is off for some reason the linter isn't catching it, but this is "correct"

Reply:
have you tried running `bin/lint` ?

Reply:
Yes I have tried but I don't think that it's auto correcting format indentation unfortunately. The issue exists before this PR, it's just apparent here because the indentation doesn't line up. Maybe erblint is misconfigured?

Reply:
```sh
bin/lint
Checking Ruby code formatting...
Checking JS code formatting...
yarn run v1.22.21
$ /Users/jm_mbp/Developer/rubyvideo/node_modules/.bin/standard --fix
✨  Done in 0.64s.
Checking erb file formatting...
Linting and autocorrecting 74 files with 17 linters (12 autocorrectable)...

No errors were found in ERB files
```